apps/web/tests/digest-playground.spec.ts

```diff
@@ -0,0 +1,185 @@
+import { test, expect } from '@playwright/test';
+
+import { getByTestId, initializeSession } from './utils.ts/browser';
+
+let session;
+
+test.beforeEach(async ({ context }) => {
+  session = await initializeSession(context, { noTemplates: true });
+});
+
+test('should have a link to the docs', async ({ page }) => {
+  await page.goto('/get-started');
+
+  const getStartedFooterLeftSide = getByTestId(page, 'get-started-footer-left-side');
+  await getStartedFooterLeftSide.click();
+
+  const tryDigestPlaygroundBtn = getByTestId(page, 'try-digest-playground-btn');
+  await tryDigestPlaygroundBtn.click();
+
+  await expect(page).toHaveURL(/\/digest-playground/);
+  await expect(page).toHaveTitle(/Digest Workflow Playground/);
```

Comment:
same for the expect code, you need to await for the results, so the Playwright under the hook will auto await for the conditions to match